src/extension.ts

```diff
@@ -158,7 +151,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
       await runServer();
     }),
     registerCommand(`${serverId}.showLogs`, async () => {
-      outputChannel.show();
+      logger.channel.show();
     }),
```

Comment:
I wonder if this is now confusing. It suggests that there's only one log and a user might not be aware that there's more than one. Should we add a new `showLspLogs` command and change their lables to `Show client logs` and `Show LSP logs`?

Reply:
Good point. I think that makes sense although what about naming it "Show language server logs" / "Show server logs" instead?

Reply:
I don't mind "Show server logs"